src/SpeedJumpIrm.sol

```diff
@@ -10,6 +10,8 @@ import {MarketParamsLib} from "../lib/morpho-blue/src/libraries/MarketParamsLib.
 import {Id, MarketParams, Market} from "../lib/morpho-blue/src/interfaces/IMorpho.sol";
 import {MathLib as MorphoMathLib} from "../lib/morpho-blue/src/libraries/MathLib.sol";
 
+int256 constant N_STEPS = 4;
```

Comment:
maybe we should say why this value?

Reply:
I can add the theoretical max errors for given days

Reply:
done